tests/plugins/widget/manual/clipboardhtmldnd.md

```diff
@@ -0,0 +1,22 @@
+@bender-tags: 4.14.0, feature, 3441
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, undo, wysiwygarea, toolbar, placeholder
+
+1. Select content inside the editor according to the instructions inside it. Drop the selected content into one of drop zones.
+
+	## Expected
+
+	After dropping widgets' content changes into `Widget copied X times!` where `X` changes after every drop. Note: the same counter is used for all widgets!
```

Comment:
It seems a bit broken when the counter is shared between widgets. Maybe we could just replace the counter with some custom message indicating that widgets have been successfully moved?